go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/vitessio/arewefastyet
+
+go 1.16
```

Comment:
I think this should be safe, but just FYI - Vitess is build on 1.15

Reply:
Should we roll back to 1.15 while https://github.com/vitessio/vitess/issues/7572 is still unresolved?